tests/test_runners/test_raw.py

```diff
@@ -148,15 +148,15 @@ def test_record_tests(self):
                     '       "status": "TEST_PASSED",',
                     '       "stdout": "This is stdout",',
                     '       "stderr": "This is stderr",',
-                    '       "createdAt": "2021-10-05T12:34:00"',
+                    '       "createdAt": "2021-10-05T12:34:00+00:00"',
```

Comment:
Do we need to update test data??

Reply:
Yes, we do.

Reply:
Why? It seems we support without time zone case
https://github.com/launchableinc/cli/pull/843/files#diff-b499b48faab08f48bbe0e94d333facabfaa86a1ef0ddfdb0460ce302b63c6996R162